src/deck-options.md

```diff
@@ -390,26 +390,26 @@ If you plug the resulting 65% into the interval modifier, you should
 find over time that your retention moves closer to your desired
 retention.
 
-One important thing to note however is that the tradeoff between time
+One important thing to note however is that the trade-off between time
 spent studying and retention is not linear: we can see here that to
-increase our retention by 5 percentage points, we’d have to study 35%
+increase our retention by 5 per cent, we would have to study 35%
```

Comment:
I'm not sure this change is correct?

https://thewritingbusiness.com/the-difference-between-percentages-and-percentage-points/

Reply:
Yes, you are right! Sorry, I think the whiskey must have run out. :)
Will you revert this change, or should I do it and resubmit a pull request?
Many thanks for you time and sorry the error!

Reply:
```suggestion
increase our retention by 5 percentage points, we would have to study 35%
```

The interface should give you an option to accept this suggested change, and then I can merge the PR.

Reply:
Nevermind, I can do that myself :-)